src/cli.ts

```diff
@@ -150,6 +150,15 @@ const build = debounce(async function build() {
       warnings.push(err);
     };
 
+    if (process.env.SOURCE_DATE_EPOCH) {
+      const sde = process.env.SOURCE_DATE_EPOCH.trim();
+      if (!/^[0-9]+/.test(sde)) {
```

Comment:
```suggestion
      if (!/^[+-]?[0-9]+/.test(sde)) {
```
i suppose :-)

Reply:
No, there's an [actual specification and everything](https://reproducible-builds.org/specs/source-date-epoch/), which says the format must be "identical to the output format of `date +%s`", which doesn't include a leading +/-.